src/Uno.UI/UI/Xaml/NativeApplication.cs

```diff
@@ -27,7 +30,16 @@ public override void OnCreate()
 
 		private void OnActivityStarted(Activity activity)
 		{
-			_app.OnLaunched(new LaunchActivatedEventArgs());
+			if (_lastHandledIntent != activity.Intent &&
```

Comment:
Why do you need to check here that the intent has changed?

Reply:
As Activity Started is called every time when app goes to foreground (for example home button and then reopen via task view), the OnLaunched with the arguments would be used each time. But the jump list item arguments are relevant only the first time